x-pack/plugins/security_solution/server/lib/detection_engine/reference_rules/ml.ts

```diff
@@ -57,6 +57,7 @@ export const mlAlertType = createSecurityMlRuleType({
     context: [{ name: 'server', description: 'the server' }],
   },
   minimumLicenseRequired: 'basic',
+  isExportable: false,
```

Comment:
Thanks, I realize this is commented out for some reason , but this should be good for when it's not.